ibft/storage/store.go

```diff
@@ -156,16 +287,14 @@ func mergeParticipants(existingParticipants, newParticipants []spectypes.Operato
 	return slices.Compact(allParticipants)
 }
 
-func (i *ibftStorage) save(txn basedb.ReadWriter, value []byte, id string, pk []byte, keyParams ...[]byte) error {
-	prefix := append(i.prefix, pk...)
-	key := i.key(id, keyParams...)
-	return i.db.Using(txn).Set(prefix, key, value)
+func (i *participantStorage) save(txn basedb.ReadWriter, value []byte, pk, slot []byte) error {
+	prefix := i.makePrefix(slot)
+	return i.db.Using(txn).Set(prefix, pk, value)
 }
 
-func (i *ibftStorage) get(txn basedb.ReadWriter, id string, pk []byte, keyParams ...[]byte) ([]byte, bool, error) {
-	prefix := append(i.prefix, pk...)
-	key := i.key(id, keyParams...)
-	obj, found, err := i.db.Using(txn).Get(prefix, key)
+func (i *participantStorage) get(txn basedb.ReadWriter, pk, slot []byte) ([]byte, bool, error) {
+	prefix := i.makePrefix(slot)
+	obj, found, err := i.db.Using(txn).Get(prefix, pk)
```

Comment:
if found is false on error then we ignore the error

since error is more significant than existance so we should check it first imo